CMakeLists.txt

```diff
@@ -586,6 +586,7 @@ if(LIBUV_BUILD_TESTS)
        test/test-udp-alloc-cb-fail.c
        test/test-udp-bind.c
        test/test-udp-connect.c
+       test/test-udp-connect6.c
```

Comment:
This also should be added to `Makefile.am`

Reply:
Done.